src/pocketdb/web/PocketExplorerRpc.cpp

```diff
@@ -52,15 +52,44 @@ namespace PocketWeb::PocketWebRpc
         return request.DbConnection()->ExplorerRepoInst->GetTransactionsStatisticByDays(topHeight, depth);
     }
 
-    UniValue GetStatisticContent(const JSONRPCRequest& request)
+    UniValue GetStatisticContentByHours(const JSONRPCRequest& request)
     {
         if (request.fHelp)
             throw std::runtime_error(
-                "getstatisticcontent\n"
-                "\nGet statistics for content transactions\n"
+                "getstatisticcontentbyhours\n"
+                "\nGet statistics for content transactions grouped by hours\n"
             );
 
-        return request.DbConnection()->ExplorerRepoInst->GetContentStatistic();
+        int topHeight = chainActive.Height() / 10 * 10;
+        if (request.params[0].isNum())
+            topHeight = std::min(request.params[0].get_int(), topHeight);
+
+        int depth = 24;
+        if (request.params[1].isNum())
+            depth = std::min(request.params[1].get_int(), depth);
+        depth = depth * 60;
+
+        return request.DbConnection()->ExplorerRepoInst->GetContentStatisticByHours(topHeight, depth);
+    }
+
+    UniValue GetStatisticContentByDays(const JSONRPCRequest& request)
+    {
+        if (request.fHelp)
+            throw std::runtime_error(
+                "getstatisticcontentbydays\n"
+                "\nGet statistics for content transactions grouped by days\n"
+            );
+
+        int topHeight = chainActive.Height() / 10 * 10;
+        if (request.params[0].isNum())
+            topHeight = std::min(request.params[0].get_int(), topHeight);
+
+        int depth = 30;
+        if (request.params[1].isNum())
+            depth = std::min(request.params[1].get_int(), depth);
+        depth = depth * 24 * 60;
```

Comment:
Multiply depth here, but divide it by same values in every place we use it after in "ExplorerRepoInst->GetContentStatisticByDays()". Does it makes sense? Same is for "GetStatisticContentByHours()"

Reply:
If I understood the question correctly, the essence of division is that in these queries we rely on the calculated index (see migration), so in queries we have to divide to match the index